demos/README.md

```diff
@@ -58,10 +58,13 @@ The table below shows the correlation between models, demos, and supported plugi
 | driver-action-recognition-adas-0002-decoder      | [Action Recognition Demo](./python_demos/action_recognition/README.md)            | Supported | Supported |             |                 |
 | driver-action-recognition-adas-0002-encoder      | [Action Recognition Demo](./python_demos/action_recognition/README.md)            | Supported | Supported |             | Supported       |
 | person-attributes-recognition-crossroad-0230     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                            | Supported | Supported | Supported   | Supported       |
+| person-attributes-recognition-crossroad-0234     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                            | Supported |           |             |                 |
+| person-attributes-recognition-crossroad-0238     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                            | Supported |           |             |                 |
 | person-reidentification-retail-0031              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                            | Supported | Supported | Supported   | Supported       |
 | person-reidentification-retail-0076              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)<br>[Multi-Camera Multi-Target Tracking Demo](./python_demos/multi_camera_multi_target_tracking/README.md)                           | Supported | Supported | Supported   | Supported       |
 | person-reidentification-retail-0079              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)<br>[Multi-Camera Multi-Target Tracking Demo](./python_demos/multi_camera_multi_target_tracking/README.md)                            | Supported | Supported | Supported   | Supported       |
 | person-vehicle-bike-detection-crossroad-0078     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                            | Supported | Supported | Supported   | Supported       |
+| person-vehicle-bike-detection-crossroad-yolov3-1016 | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                         | Supported |           |             |                 |
```

Comment:
```suggestion
| person-vehicle-bike-detection-crossroad-1016 | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                         | Supported |           |             |                 |
```

Reply:
I can see `person-vehicle-bike-detection-crossroad-yolov3-*` exist but with different numbers. And there is `person-vehicle-bike-detection-crossroad-1016`. But I can't find a model which has yolov3 suffix and 1016 number.

Yolo usually has multiple [outputs](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md#outputs), while the demo requires a detection model to have only one output.

Reply:
Oh, I see. Fixed